### ISSUE TITLE
Extraneous `vendor/eh` directory in distribution

### ISSUE THREAD

**Issue by kislyuk** [dtkav/swagger_ui_bundle#5]
There is a directory, `vendor/eh`, in the files linked from https://pypi.org/project/swagger-ui-bundle/#files (https://files.pythonhosted.org/packages/a4/73/7e2f7e104586277a6838543a600bf837a52d052b1341e1661f1f120eaa25/swagger_ui_bundle-0.0.4-py3-none-any.whl and https://files.pythonhosted.org/packages/28/f3/21c81652f03115c3c6e28a643bc5c7a83963facc785dadded3084f19cb49/swagger_ui_bundle-0.0.4.tar.gz), that contains the following:
```
swagger-ui-3.22.3
swagger-ui-3.22.3.tar.gz
```
Diffing the contents, it looks like this is a staging directory for the swagger-ui distribution:
```
>diff -ur vendor/swagger-ui-3.22.3 vendor/eh/swagger-ui-3.22.3
Only in vendor/eh/swagger-ui-3.22.3: .agignore
Only in vendor/eh/swagger-ui-3.22.3: .babelrc
Only in vendor/eh/swagger-ui-3.22.3: .dockerignore
Only in vendor/eh/swagger-ui-3.22.3: .editorconfig
Only in vendor/eh/swagger-ui-3.22.3: .eslintrc
Only in vendor/eh/swagger-ui-3.22.3: .gitattributes
Only in vendor/eh/swagger-ui-3.22.3: .github
Only in vendor/eh/swagger-ui-3.22.3: .gitignore
Only in vendor/eh/swagger-ui-3.22.3: .npmignore
Only in vendor/eh/swagger-ui-3.22.3: CONTRIBUTING.md
Only in vendor/eh/swagger-ui-3.22.3: Dockerfile
Only in vendor/eh/swagger-ui-3.22.3: LICENSE
Only in vendor/eh/swagger-ui-3.22.3: README.md
Only in vendor/eh/swagger-ui-3.22.3: composer.json
Only in vendor/eh/swagger-ui-3.22.3: cypress.json
Only in vendor/eh/swagger-ui-3.22.3: dev-helpers
Only in vendor/eh/swagger-ui-3.22.3: dist
Only in vendor/eh/swagger-ui-3.22.3: docker
Only in vendor/eh/swagger-ui-3.22.3: docs
Only in vendor/eh/swagger-ui-3.22.3: flavors
Only in vendor/swagger-ui-3.22.3: index.j2
Only in vendor/eh/swagger-ui-3.22.3: make-webpack-config.js
Only in vendor/eh/swagger-ui-3.22.3: package-lock.json
Only in vendor/eh/swagger-ui-3.22.3: package.json
Only in vendor/eh/swagger-ui-3.22.3: postcss.config.js
Only in vendor/eh/swagger-ui-3.22.3: release
Only in vendor/eh/swagger-ui-3.22.3: snapcraft.yaml
Only in vendor/eh/swagger-ui-3.22.3: src
Only in vendor/eh/swagger-ui-3.22.3: swagger-config.yaml
Only in vendor/eh/swagger-ui-3.22.3: swagger-ui-dist-package
Only in vendor/eh/swagger-ui-3.22.3: test
Only in vendor/eh/swagger-ui-3.22.3: webpack-dist-bundle.config.js
Only in vendor/eh/swagger-ui-3.22.3: webpack-dist-standalone.config.js
Only in vendor/eh/swagger-ui-3.22.3: webpack-dist.config.js
Only in vendor/eh/swagger-ui-3.22.3: webpack-hot-dev-server.config.js
Only in vendor/eh/swagger-ui-3.22.3: webpack-watch.config.js
Only in vendor/eh/swagger-ui-3.22.3: webpack.check.js
Only in vendor/eh/swagger-ui-3.22.3: webpack.config.js
Only in vendor/eh/swagger-ui-3.22.3: webpack.dist-style.config.js
```
Why is this directory in the release? My application relies on minimizing the size of this dependency - can this package be released without this staging directory? Thanks.

**Comment by dtkav** [comment]
Thanks @kislyuk - good catch.
That was my mistake in the package upload step. 
I'll bump the version and do the upload properly this time...
My apologies for the inconvenience.

**Comment by dtkav** [comment]
This should now be resolved with v0.0.5.
Please reopen if I missed something.

**Comment by kislyuk** [comment]
Thanks @dtkav, I can confirm that the extra directory is gone.
